src/builds/builds.service.ts

```diff
@@ -29,6 +29,19 @@ export class BuildsService {
     });
   }
 
+  async deleteOldBuilds(projectId: string, keepBuilds: number) {
+    const [total, buildList] = await Promise.all([
```

Comment:
by passing `undefined` in `findMany` method you could take the rest builds after skipping `keepBuilds`